property.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS `properties` (
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(255) NOT NULL,
+	`interior` VARCHAR(255) NOT NULL,
+	`furnished` BOOLEAN NOT NULL,
+	`owners` JSON,
+	`garage` BOOLEAN NOT NULL,
+	`coords` JSON NOT NULL,
+	`price` INT NOT NULL,
+	`rent` INT NOT NULL,
+	`rent_date` TIMESTAMP DEFAULT NULL,
+	`stash` TEXT,
+	`outfit` TEXT,
+	`logout` TEXT,
+	`decorations` INT,
+	`garage_slots` JSON,
+	PRIMARY KEY (id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS `property_owners` (
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`property_id` INT NOT NULL,
+	`citizenid` INT NOT NULL,

```

Comment:
citizenid isn't a foreign key?

Reply:
Should be good now with the Foreign key thing? let me know tho